rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -284,3 +287,37 @@ def rate_limit_write(bucket, max_write_ops, ssl=None):
     )
     stdout, stderr = run_subprocess(cmd)
     assert "503" in str(stderr), "Rate limit slowdown not observed, failing!"
+
+
+def test_full_sync_at_archive(bucket_name):
+    """
+    test_full_sync_at_archive zone for a bucket
+    """
+    zone_name = "archive"
+    bucket_name = f"tenant/{bucket_name}"
+    cmd_bucket_stats = f"radosgw-admin bucket stats --bucket {bucket_name}"
+    log.info(f"collect bucket stats for {bucket_name} at local site")
+    local_bucket_stats = json.loads(utils.exec_shell_cmd(cmd_bucket_stats))
+    local_num_objects = local_bucket_stats["usage"]["rgw.main"]["num_objects"]
+    local_size = local_bucket_stats["usage"]["rgw.main"]["size"]
```

Comment:
@viduship , could you create a method to get the bucket stats at local & remote in the s3cmd/reusable?

Reply:
alright.

Reply:
done , 
latest logs http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-OL8YPW